Resources/Prototypes/Entities/Structures/Dispensers/soda.yml

```
@@ -50,3 +51,6 @@
     - id: DrinkTeaJug
     - id: DrinkTonicWaterBottleFull
     - id: DrinkWaterMelonJuiceJug
+# Moffstation Start - Dispenser QOL changes
+    - id: DrinkGrenadineBottleFull
+# Moffstation End
```

```suggestion
    - id: DrinkGrenadineBottleFull # Moffstation - Dispenser QOL changes
```
Single line modifications get single line guards, not block guards.

---

makes sense